pagopa/io-app
# chore(EU Covid Certificate): [IAGP-10,IAGP-12] Update Message spec with EUCovidCert

## Short description
This PR updates the message spec to include `eu_covid_cert` section

## List of changes proposed in this pull request
- add eu_covid_cert type guard
- add tests

## How to test
- make sure to have the last commit of` io-dev-server` (`master` branch)
- use this snippet to test messages
    ![Schermata 2021-06-03 alle 15 21 49](https://user-images.githubusercontent.com/822471/120652299-f083eb00-c47f-11eb-8260-235fa1ce270e.png)
